examples/threejs_worker.js

```diff
@@ -169,8 +169,8 @@ function start(video, input_width, input_height, render_update, track_update) {
 
   var process = function () {
     context_process.fillStyle = 'black';
-    context_process.fillRect(0, 0, pw*2, ph*2);
-    context_process.drawImage(video, 0, 0, vw, vh, ox, oy, w*2, h*2);
```

Comment:
same as above, and it wwasn't correct because width and height in getImageData wasn't correct (they should be double but they wasn't...)